examples/fabric/nested_ingestion_pipeline/README.md

```diff
@@ -0,0 +1,96 @@
+# Nested pipelines
+
+The following example, exemplifies the utilization of nested pipelines including control activities like ForEach and If.
```

Comment:
Perhaps we can start this page with a short description of what the whole pipeline is trying to achieve. Otherwise the reader has to reverse engineer that from the pipeline and description itself and I am struggling to reverse engineer it myself.

Example: `This pipeline is an example that reads the configuration of a lakehouse, then checks which files are newer or updated and then only downloads those files, uploads them to ADLS and eventually into the Lakehouse.` <- Probably incorrect

Reply:
Thanks, that helps!